hyperlink/test/common.py

```diff
@@ -0,0 +1,70 @@
+
+
+from unittest import TestCase
+
+
+class HyperlinkTestCase(TestCase):
```

Comment:
I'd love to use py.test directly. This is more an artifact of Twisted's approach/Moshe's recommendation that I write in unittest and run with a more modern test runner like trial/py.test.

I figured I'd handle that as a separate refactor, separate the concerns a bit.

As far as 2.6 compat, well, it's easy. The code itself needed no changes, only the tests. And as someone who's working days porting a codebase off 2.6, I certainly appreciate it. :)